model/sort.go

```diff
@@ -18,6 +18,24 @@ import (
 	"sort"
 )
 
+type byTraceID []*TraceID
+
+func (s byTraceID) Len() int      { return len(s) }
+func (s byTraceID) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
+func (s byTraceID) Less(i, j int) bool {
+	if s[i].High < s[j].High {
+		return true
+	} else if s[i].High > s[j].High {
+		return false
+	}
+	return s[i].Low < s[j].Low
+}
+
+// SortTraceIDs sorts a list of TraceIDs
+func SortTraceIDs(traceIDs []*TraceID) {
+	sort.Sort(byTraceID(traceIDs))
+}
```

Comment:
This does not seem general purpose enough to be in the `model`.

Reply:
If there's ``SortTraces``, ``SortTrace``, ``SortSpan`` then why wouldn't ``SortTraceID`` be proper? Feels logical to me.